src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+	private Tag tag;
+	private Person person;
+	
+	/** 
+	 * If true, this tag was added and a '+' will be printed.
+	 * If false, this tag was deleted and a '-' will be printed. 
+	 */
+	private boolean isAddedTag;
```

Comment:
As discussed in class, boolean may not be the most appropriate data type to use here.